src/codesyncd/handlers/tab_handler.ts

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
+import path from "path";
+
+import { ITabYML } from "../../interface";
+import {generateSettings} from "../../settings";
+import { removeTabFile } from "../../utils/tab_utils";
+import { isRelativePath } from "../utils";
+
+export class TabHandler {
+	
+	constructor() {}
```

Comment:
If the constructor is empty, should we consider defining `static` methods?

Reply:
No we don't. Removing it